src/cozmo/conn.py

```diff
@@ -372,10 +373,16 @@ def _handle_ui_device_connected(self, msg):
                 'cozmoclad_version=%s app_build_version=%s',
                 version.__version__, cozmoclad.__version__, msg.buildVersion)
 
-        # We send RequestObjectStates before refreshing the animation names
-        # as this ensures that we will receive the responses before we mark the
-        # robot as ready
-        msg = _clad_to_engine_iface.RequestObjectStates()
+        # We send RequestConnectedObjects and RequestLocatedObjectStates before
+        # refreshing the animation names as this ensures that we will receive
+        # the responses before we mark the robot as ready
```

Comment:
Let's add a period after "ready" and maybe a blank line to separate this from the next line.